mlir/utils/performance/tuningRunner.py

```diff
@@ -329,15 +352,27 @@ def main(args=None):
         return
 
     if parsed_args.debug:
-        print(allData)
+        print(allData, file=sys.stderr)
         allData.to_csv(f"{parsed_args.output}.debug", sep='\t')
 
     # Note, appending results here to allow multiple config sets
-    with open(parsed_args.output, 'a') as outFile:
-        print(f"# arch\tnumCUs\ttestVector\tperfConfig ({options.tuningSpaceKind})", file=outFile)
-        for testVector, perfConfig in winners.items():
-            print(f"Arch = {arch}({numCU} CUs), vector = '{testVector}', perfConfig = {perfConfig}")
-            print(f"{arch}\t{numCU}\t{testVector}\t{perfConfig}", file=outFile)
+    if parsed_args.output == '-':
```

Comment:
Writing results to stdout is another way to avoid collisions when multithreaded -- there's no file, thus no chance of identical filenames.